src/pages/wishlist/Wishlist.js

```diff
@@ -0,0 +1,43 @@
+import { useState } from "react";
+import Card from "../../components/Card/Card";
+import FilterSideBar from "../../components/FilterSideBar/FilterSideBar";
+import Navbar from "../../components/Navbar/Navbar";
+
```

Comment:
you can create an index.js file in components and re-export all components from there.